cmd/executor/cmd/root.go

```diff
@@ -223,6 +223,7 @@ func resolveDockerfilePath() error {
 			return errors.Wrap(err, "getting absolute path for dockerfile")
 		}
 		opts.DockerfilePath = abs
+		opts.OriginalDockerfilePath = abs
```

Comment:
Is there a difference between `OriginalDockerfilePath` and `DockerfilePath`

Reply:
Yes because the line immediately following in copyDockerFile

https://github.com/GoogleContainerTools/kaniko/blob/0477900febfacb98b88abdbfaf8bbaeb05f51060/cmd/executor/cmd/root.go#L257

Reply:
Sorry @cehoffman for the delayed response. The L257 code makes sure, we fall back to `Dockerfile` if the file passed in command line arg `--dockerfile` does not exist. 

If we fallback to `Dockerfile` then the dockerfile ignore should also be `Dockerfile.dockerignore` right?

Reply:
L257 I believe is to make the Dockerfile used for build always ends up at `/kaniko/Dockerfile`. I see now I didn’t modify the the context local Dockerfile lookup on L235

I think perhaps the right solution is to extend `copyDockerfile()` to detect a relative dockerignore and copy it to `/kaniko/Dockerfile.dockerignore`. That would then let the existing lookup resolve and not require the `OriginalDockerfilePath` struct item.